rosdep/python.yaml

```diff
@@ -5938,6 +5938,7 @@ python3-ruamel.yaml:
   ubuntu:
     artful: [python3-ruamel.yaml]
     bionic: [python3-ruamel.yaml]
+    focal: [python3-ruamel.yaml]
     xenial: [python3-ruamel.yaml]
     yakkety: [python3-ruamel.yaml]
     zesty: [python3-ruamel.yaml]
```

Comment:
Given that this is supported on all of our supported platforms with the same name, I'll suggest changing this to:

```
ubuntu: [python3-ruamel.yaml]
```